informer/informer_integration_basic_test.go

```diff
@@ -73,9 +75,9 @@ func Test_Informer_Integration_Basic(t *testing.T) {
 			testAssertCROWithID(t, e, objectIDTwo)
 		}
 
-		d := time.Now().Sub(start)
-		if d.Seconds() > 0.1 {
-			t.Fatalf("expected %#v got %#v", "round about 0.1 second", d.Seconds())
+		d := time.Since(start)
+		if !durationEquals(0, d, timeDelta) {
+			t.Fatalf("expected %#v got %#v", "round about 0 seconds", d.Seconds())
 		}
```

Comment:
Ninja fix: now we assert time is +/- timeDelta. I extracted timeDelta const so we can easy change it for slow CI servers in the future.